src/modules/m_nick.c

```diff
@@ -333,6 +332,7 @@ void nick_identify (Nick *nptr, User *uptr __unused, char *all)
     NoticeToUser(nptr,"You are now identified");
     nptr->loginattempts = 0;
     nptr->lasttry = 0;
+    nptr->svid = user->nick;
```

Comment:
This should not compile, as you're reassigning an array. What you want is:

strncpy(nptr->svid, user->nick, SVIDLEN);

Reply:
Old copy, fixed in the branch already (though I used SVIDLEN + 1, will update that).